mopidy/mpd/actor.py

```diff
@@ -57,9 +60,7 @@ def on_stop(self):
         process.stop_actors_by_class(session.MpdSession)
 
```

Comment:
This probably doesn't make sense, reasoning being that we need to add playlists_loaded, playlist_changed, playlist_deleted handling which will update the URI mapper based on changes.